ingredient/ingredient-container-reg/src/functions.ts

```diff
@@ -0,0 +1,23 @@
+import {BaseUtility, IngredientManager} from '@azbake/core'
+
+export class ContainerRegUtils extends BaseUtility {
+
+    public get_profile(): string {
```

Comment:
Rename to create_resource_name(), which I believe is the standard pattern for this.

Reply:
Renamed to `create_resource_name()`